core/src/main/java/com/linecorp/armeria/internal/server/FileAggregatedMultipart.java

```diff
@@ -61,15 +64,29 @@ public ListMultimap<String, MultipartFile> files() {
         return files;
     }
 
+    private static boolean shouldHandle(BodyPart bodyPart, List<String> parameters) {
+        final String name = bodyPart.name();
+        assert name != null;
+        if (parameters.isEmpty()) {
```

Comment:
If an annotated service doesn't have any parameters, I think we'll run into the same issue.

What do you think of just accepting a `Multipart` to avoid ambiguity?

e.g.
```java FileAggregatedMultipart.java
    public static CompletableFuture<FileAggregatedMultipart> aggregateMultipart(
            ServiceRequestContext ctx, HttpRequest req) {
        final Multipart multipart = Multipart.from(req);
        return aggregateMultipart(ctx, multipart);
    }

    public static CompletableFuture<FileAggregatedMultipart> aggregateMultipart(
            ServiceRequestContext ctx, Multipart multipart) {
...
```

and then, we can pass in the filtered `Multipart` from `AnnotatedService`

```java DefaultAnnotatedService.java
...
    private CompletionStage<HttpResponse> serve1(ServiceRequestContext ctx, HttpRequest req,
                                                 AggregationType aggregationType) {
        final CompletableFuture<AggregatedResult> f;
        switch (aggregationType) {
            case MULTIPART:
                f = FileAggregatedMultipart.aggregateMultipart(ctx, Multipart.from(req).filter(bodyPart -> shouldHandle(bodyPart, parameters)))
                                           .thenApply(AggregatedResult::new);
...
```